products/mozilla_org/extract_strings.md

```diff
@@ -0,0 +1,79 @@
+
+Initial review should include checking the file against the page on the testing server and checking against the template. This is to ensure that there are no hard coded strings.
+
+Note: if there is an update to an existing page, it is advised to still generate the .lang file, and copy the new string there instead of copying it from the .html file. This is to make sure, for example, that the syntax used for variable replacements is correct, and remove eventual extra white spaces between words that are not carried over to the .lang file.
+
+### Add notes and comments to the extracted file:
+
+Add important information at the top of the file:
+  `## NOTE: this is a note.`
+* A Note for [demo/test URLs](https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/facebookcontainer/index.lang).
+* A Note for test instruction if necessary, for a page that is more dynamic, such as the [privacy_protection_tour.lang](https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/tracking-protection-tour.lang).
+* Make sure to add tags, if there are strings [bound to l10n tags](working_bedrock.md#l10n-tags).
+
+Proceed to the content of the file. Add a comment to the following type of strings to provide more context:
+* Page title and description. Typically, a page title string should be translated in a short or catchy way
+* If the string contains a link, to which [URL](https://github.com/mozilla-l10n/vm_scripts/blob/master/update_mozillaorg.sh0 it points to. This gives context to localizers.
+* If a string is behind a tag, make sure to add the [tag binding](https://github.com/mozilla-l10n/langchecker/wiki/.lang-files-format#string-meta-data).
+* A new string behind a tag makes another string obsolete. Make sure to add [an obsolete comment](https://github.com/mozilla-l10n/vm_scripts/blob/master/update_mozillaorg.sh) to help localizers prioritize their work, and that old and new strings are next to each other. This will also make removal of old strings easier in further updates.
+* A comment for an alternative to an expression that may not be culturally relevant, or to a pun that doesn’t translate well.
+* Check English content for capitalization, hyphenated word to make sure they are consistent with the US English style guide (macOS vs. MacOS, sign in to vs. sign into).
+
+When all is done, create a PR in the www.mozilla.org repo for review. Describe what the PR is for, add references to the Bedrock’s pull requests to add context for the review.
+
+Check for possible [errors](https://l10n.mozilla-community.org/langchecker/?action=errors) such as duplicates or unmatched strings before requesting for review. In order to do this, you will need to run langchecker locally, and make sure that both the l10n repository and langchecker’s are using the branches you’re working on.
```

Comment:
Again: Langchecker when it's in the middle of a sentence. The last section is full of them.